### PR TITLE
Add file extensions for Markdown

### LINKED ISSUE #8: Additional file extensions
Hello, `rubocop-md` currently supports `.md` and `.markdown` extensions,

https://github.com/rubocop-hq/rubocop-md/blob/38addb370ca65283ce9d8d9770d6fe6ce64c65b7/lib/rubocop/markdown/rubocop_ext.rb#L5

also, what about adding the support of the following extensions according to `linguist`?

https://github.com/github/linguist/blob/96ca71ab99c2f9928d5d69f4c08fd2a51440d045/lib/linguist/languages.yml#L3065-L3083

- .mdown
- .mdwn
- .mdx
- .mkd
- .mkdn
- .mkdown
- .ronn
- .workbook

### PR DESCRIPTION
**Pull request by ybiquitous**
Fix #8

### CODE REVIEW

**Comment by bbatsov** [comment]
The changes look good to me. Don't forget to update the changelog as well.

**Comment by ybiquitous** [comment]
@bbatsov Thanks for your feedback. I've updated the changelog by adding this PR title and my GitHub account (see e628ae7).

In addition, I want to add a link to this PR like [the changelog on rubocop-hq/rubocop](https://github.com/rubocop-hq/rubocop/blob/master/CHANGELOG.md).
But I keep the changelog format's consistency on e628ae7.

What do you think about adding a link to a related PR?

File: lib/rubocop/markdown/rubocop_ext.rb [hunk 1]
```diff
@@ -2,7 +2,20 @@
 
 module RuboCop
   module Markdown # :nodoc:
-    MARKDOWN_EXTENSIONS = %w[.md .markdown].freeze
+    # According to Linguist.
+    # See https://github.com/github/linguist/blob/96ca71ab99c2f9928d5d69f4c08fd2a51440d045/lib/linguist/languages.yml#L3065-L3083
```
Comment (palkan) [inline lib/rubocop/markdown/rubocop_ext.rb]: 👍

**Comment by ybiquitous** [comment]
@palkan @bbatsov Thanks for your review!

**Comment by palkan** [comment]
Thanks for your work! I gonna release it as 0.3.2 later today.

**Pull request merged**